internal/provider/resources/install/azure/azure.go

```diff
@@ -30,31 +30,42 @@ type Azure struct {
 }
 
 type azureModel struct {
-	ClientId            types.String `tfsdk:"client_id"`
 	DirectoryId         types.String `tfsdk:"tenant_id"`
```

Comment:
nit: can we rename this to directory_id? or is this set by the backend and is that why we can't change this? 🤔